src/api/json/catalog.json

```diff
@@ -1617,11 +1617,12 @@
     },
     {
       "name": "ui5-manifest",
-      "description": "UI5/OPENUI5 descriptor file",
+      "description": "UI5 manifest file",
```

Comment:
The URL looks good. So, we always are at the latest state.

Maybe we should leave the `.manifest` schema untouched and keep the reference to mikeb's schema here. We should maybe also ask how and where the `.manifest` is used. In our case it is not considered at all.

If we plan to consolidate it, we should also remove the schema from mikeb in the repo. 

Right now we could also just have a separate section with filematch "manifest.json" - maybe there are some more file matches for this file and this might conflict. In the description I would use `UI5 manifest (manifest.json)`. This is a bit more aligned with other entries I checked...

Reply:
thanks a lot for teh feedback, please see the file after my last commit